app/src/main/res/layout/activity_main.xml

```diff
@@ -9,7 +9,7 @@
     <TextView
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
+        android:text="hogefuga"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/WataruSuzuki/Android-Example-Using-Danger/blob/0c040d149df52a583eb7528530ec4b05dfa97953/app/src/main/res/layout/activity_main.xml#L12"></span>Hardcoded string “hogefuga”, should use <code>@string</code> resource</td>
    </tr>
  </tbody>
</table>